ert_gui/tools/plot/storage_client.py

```diff
@@ -37,19 +51,23 @@ def all_data_type_keys(self):
         ]
         """
 
-        r = requests.get("{base}/ensembles".format(base=self._BASE_URI))
+        ensembles = ref_request("{base}/ensembles".format(base=self._BASE_URI))
+        ens_schema = ref_request(ensembles["ensembles"][0]["ref_url"])
 
-        ens_pointer = r.json()["ensembles"][0]["ref_url"]
+        def obs_for_response(ref_url):
+            response = ref_request(ref_url)
 
-        r = requests.get(ens_pointer)
+            if "observation" not in response:
+                return {}
 
-        ens_schema = r.json()
+            response["observation"]["name"] = response["name"]
+            return response["observation"]
```

Comment:
I was a bit confused about the structure of `observation` as it didn't seem to allow for more than a single observation be added to a response.

In `snake_oil` that is the case for `WOPR:OP1`, which has multiple observations attached:

```
SUMMARY_OBSERVATION WOPR_OP1_9
{
    VALUE   = 0.1;
    ERROR   = 0.05;
    RESTART = 9;
    KEY     = WOPR:OP1;
};

SUMMARY_OBSERVATION WOPR_OP1_36
{
    VALUE   = 0.7;
    ERROR   = 0.07;
    RESTART = 36;
    KEY     = WOPR:OP1;
};
....
```

If I launch the flask server and go to: `http://127.0.0.1:5000/ensembles/1/responses/WOPR:OP1`, I don't get more than one observation.
```
  "name": "WOPR:OP1", 
  "observation": {
    "data": {
      "data_indexes": {
        "data_url": "http://127.0.0.1:5000/data/18"
      }, 
      "key_indexes": {
        "data_url": "http://127.0.0.1:5000/data/17"
      }, 
      "std": {
        "data_url": "http://127.0.0.1:5000/data/20"
      }, 
      "values": {
        "data_url": "http://127.0.0.1:5000/data/19"
      }
    }
  },
```

Just to verify, the old gui does include all observations when plotting.

Reply:
Apparently fixed in the latest version of the branch!